environments.yaml.tmpl

```diff
@@ -1,10 +1,19 @@
 environments:
- default:
-   values:
-   - ../etc/base.yaml
-   - ../etc/production.yaml
-   - ../etc/production.yaml.gotmpl
-   - ../etc/secrets.yaml
+  default:
+    values:
+    - ../etc/base.yaml
+    - ../etc/production.yaml
+    - ../etc/production.yaml.gotmpl
+    - ../etc/secrets.yaml
+   {{ if .Values.disable_tls }}
+   - ../etc/_mods/disable_tls.yaml
+   {{ end }}
+   {{ if .Values.dev_deployment }}
+   - ../etc/_mods/disable_tls.yaml
+   - ../etc/_mods/disable_logging_and_monitoring.yaml
```

Comment:
I think there should be a separate flag for disabling logging and monitoring and it should be true.

Reply:
I disabled logging and monitoring in the default deployment. We need to be careful here, because this change is not backwards compatible. We should provide a migration note here?

Reply:
Yes it's good to mention in release notes to manually enable these components during upgrade.

Reply:
I also created a separate flag to disable monitoring and logging.